src/code/z_actor.c

```diff
@@ -2707,19 +2708,86 @@ void Actor_DrawLensActors(PlayState* play, s32 numInvisibleActors, Actor** invis
     CLOSE_DISPS(gfxCtx, "../z_actor.c", 6284);
 }
 
-s32 func_800314B0(PlayState* play, Actor* actor) {
-    return func_800314D4(play, actor, &actor->projectedPos, actor->projectedW);
+/**
+ * Checks if an actor should be culled or not, by seeing if it is contained within it's own culling volume.
+ * For more details on the culling test, see `Actor_CullingVolumeTest`.
+ *
+ * "Culling" in this context refers to the removal of something for the sake of improving performance.
+ * For actors, being culled means that their Update and Draw processes are halted.
+ * When an actor is culled, their update state is frozen and they will not draw, so they will be visible.
```

Comment:
This is kind of redundant, they are essentially saying the same thing. One of the 2 lines should be picked, and if the second one is chosen, it should be `invisible` or `they won't be visible` as I think that's the intended meaning

Reply:
I dont see them as redundant personally, the first is a general statement "these things are halted". And the followup is describing exactly what happens when those processes are halted (they are frozen and invisible)

It is meant to say invisible tho, yeah

Reply:
I can change the lead in for the second sentence, so it is more apparent that it is a clarification:
```
 * For actors, being culled means that their Update and Draw processes are halted.
 * This means that their update state is frozen and they will not draw, so they will be invisible.
 ```

Reply:
I guess just reading it myself I read it as "being culled means update and draw are halted, and when they're culled, they are frozen and invisible" which to me, that first line is saying implicitly. Maybe I would more expect something like 

"For actors, being culled means that their Update and Draw processes are halted. While halted, their update state is frozen and they will not draw, making them invisible."

Though I suppose I'm not reading it as a layman and as someone who actually knows what is going on. More than likely, ignore this and just fix the invisible part lmao

Reply:
Im fine with your suggestion